service/src/main/kotlin/no/nav/su/se/bakover/service/sak/SakServiceImpl.kt

```diff
@@ -165,14 +166,20 @@ class SakServiceImpl(
         }
     }
 
-    override fun lagreOgSendFritekstDokument(request: OpprettDokumentRequest): Either<KunneIkkeOppretteDokument, Dokument.MedMetadata> {
+    override fun genererLagreOgSendFritekstDokument(request: OpprettDokumentRequest): Either<KunneIkkeOppretteDokument, Dokument.MedMetadata> {
         return opprettFritekstDokument(request).map {
             it.leggTilMetadata(Dokument.Metadata(sakId = request.sakId), request.distribueringsadresse)
         }.onRight {
             dokumentRepo.lagre(it)
         }
     }
 
+    override fun lagreOgSendFritekstDokument(request: JournalførOgSendDokumentCommand): Dokument.MedMetadata {
+        return request.opprettDokumentMedMetadata(clock).also {
+            dokumentRepo.lagre(it)
```

Comment:
Her tar vi for god fisk at sakId finnes, og lar dokument sin foreign key til sak-tabellen smelle hvis sakId ikke stemmer :shrug: Burde kanskje legge inn som en kommentar. Domenet bør egentlig ha kontroll her selv.